mwrpy/level1/quality_control.py

```diff
@@ -7,6 +7,7 @@
 import pandas as pd
 import suncalc
 from numpy import ma
+from PIL.ImageChops import offset
```

Comment:
Not used?

Reply:
Not sure where this comes from.. Can be removed.